environment.devenv.yml

```diff
@@ -17,7 +17,11 @@ dependencies:
     - cmake >=3.5.2
 
     {% if os.sys.platform != 'win32' %}
-    - clangdev>=6.0.0
+    - compilers
+    - gxx_impl_linux-64
+    - gcc_impl_linux-64
+    - gxx_linux-64
+    - gcc_linux-64
```

Comment:
@tadeu @arthursoprana I need some help here. The thing is: conda env breaks if we use `compilers` only. Then I took a look at `eden` and included here a similar stack for C/Cpp. However, in Travis, the compilers are gathered from the system instead of the env. Please take a look at the snippet below:

```console
-- The CXX compiler identification is GNU 5.4.0
-- The C compiler identification is GNU 5.4.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
```

Is this OK? What do you recommend?

Reply:
No, it should use the compiler from the environment. Probably the line `set(CMAKE_CXX_COMPILER    c++)` (etc.) is messing up, it's trying to find `c++` instead of what `conda` defines in the `CXX` env var (and the `CC` env var).

Reply:
Ok, I will try to seek a way to fix this problem

Reply:
@tadeu fixed. The thing is: the CI test is performed within an env created by `tox`. However, `conda_deps` inside `tox.ini` doesn't include C/C++ compilers according to `conda`. So, inside `tox` env, there is no compiler, therefore the compilers will be gathered from system. I had to change `tox.ini` to include a minimal set of compilers for C and C++. I'm aware about `compilers` from `conda`, they are intend to be used for builds, but I think that we are OK using such package for tests with `tox` here.